sphinx/transforms/i18n.py

```diff
@@ -258,20 +258,19 @@ def is_named_target(node):
                     (nodes.paragraph,) + LITERAL_TYPE_NODES + IMAGE_TYPE_NODES):
                 continue  # skip for now
 
-            # auto-numbered foot note reference should use original 'ids'.
```

Comment:
This block is only for auto-numbered footnote_refs. So please don't merge standard footnote_refs here.